isodesign/io.py

```diff
@@ -20,8 +20,56 @@
 
 """
 
+import pandas as pd 
+from pathlib import Path
 
+class IoGestion:
+    def __init__(self):
+        self.netw = None
+        self.tvar = None
+        self.mflux = None
 
+    def read_file(self, data : str):
+        try:
+            if isinstance(data, str):
+                data_path = Path(data).resolve()
+                if data_path.exists(): 
+                    if data_path.suffix in [".netw"]:
+                        self.netw = pd.read_csv(data, sep='\t', skiprows=[0], header=None)
+                    elif data_path.suffix in [".tvar"]:
+                        self.tvar = pd.read_csv(data, sep='\t')
+                    elif data_path.suffix in [".mflux"]:
+                        self.mflux = pd.read_csv(data, sep='\t', skiprows=[0])
+                    else:
+                        raise TypeError (f"{data_path} is not in the good format\n Only .netw, .tvar, .mflux formats are accepted")
+                else:
+                    raise ValueError(f"{data_path} doesn't exist.")
+            else:
+                raise TypeError(f"An input file is not in right type.")
```

Comment:
L'erreur ici devrait plutôt être:
`f"{data} should be of type string and not {type(data)}"`